game_of_life.py

```diff
@@ -77,6 +78,10 @@ def create_game(grid_size: Optional[Tuple[int, int]] = (20, 20), max_iter: Optio
             elif not game_started and event.type == pygame.KEYDOWN:
                 game_started = True
                 print("Starting game of life... 🖥️🕹️🐍")
+            elif game_started and event.type == pygame.KEYDOWN:
```

Comment:
Does it mean the game will take multiple screenshots each time there is a key press? What about assigning controls (like `s` for screenshot, `p` trigger the game)

Reply:
> Does it mean the game will take multiple screenshots each time there is a key press? 

Yes, once the game is started any key triggers a screenshot

> What about assigning controls (like s for screenshot, p trigger the game)

That's a good idea :+1: